back/tests/cmd/openbsrv-integration/main.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/OpenEugene/openboard/back/internal/pb"
+	"github.com/codemodus/relay"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	userServiceTests()
+	//postServiceTests()
+}
+
+func userServiceTests() {
+	check, ff := relay.New().Fns()
+	defer func() { ff(recover()) }()
+
+	cc, err := grpc.Dial(":4242", grpc.WithInsecure())
+	check(err)
+	defer cc.Close()
+
+	c := pb.NewUserSvcClient(cc)
+
+	r1, err := c.AddRole(
```

Comment:
I'm receiving an error:
```
openbsrv-integration: rpc error: code = Unknown desc = Error 1265: Data truncated for column 'role_id' at row 1
```
Have you been sure to rollback/migrate after any changes that affect the database? If so, are you doing something non-obvious that might make this error not occur for you?

Reply:
This is happening because the database will not accept the string IDs, I think. The database wants the integer IDs. I don't think these tests will run until I fix the migration to alter the columns so they accept varchar values.